tests/unit/style/stylesheets.rs

```diff
@@ -347,4 +354,45 @@ fn test_report_error_stylesheet() {
     assert_eq!("Unsupported property declaration: 'display: invalid;'", error.message);
     assert_eq!(4, error.line);
     assert_eq!(9, error.column);
+
+    // testing for the url
+    assert_eq!(url, error.url);
+    // assert_eq!(wrong_url, error.url);
+}
+
+
+#[test]
+fn test_report_error_passing_correct_url() {
```

Comment:
added a new test for checking urls
loop thru different urls and check if they are being passed correctly